core/commonMain/src/kotlinx/serialization/internal/CollectionSerializers.kt

```diff
@@ -70,13 +70,13 @@ internal sealed class ListLikeSerializer<Element, Collection, Builder>(
         }
     }
 
-    protected final override fun readAll(decoder: CompositeDecoder, builder: Builder, startIndex: Int, size: Int) {
+    final override fun readAll(decoder: CompositeDecoder, builder: Builder, startIndex: Int, size: Int) {
```

Comment:
Any reason you remove the protected visibility from these functions? They are internal and none of your changes seem to reference them in a way incompatible with protected. This is the same for most other removals of protected.

Reply:
Please see the commit message title for this commit. This is just a cleanup of redundant visibility modifiers reported by IDE inspection.